### DISCUSSION TITLE
Would this work with these wafer recessed lights?

### DISCUSSION THREAD

**Discussion by tcozzens** [milo526/homebridge-tuya-web#335] [Q&A]
Hi there, 

The app used for these is smart life, I was just curious if there's a way to test that they would work with this plugin?

https://www.amazon.com/gp/product/B098J2ZLMW/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

**Comment by milo526** [comment]
Most of the lights that work with the smart life app will work with this plugin.
There is however no guarantee possible short of testing it.
So either somebody who owns these lights will tell you that they work or you'll have to try for yourself.

I personally do not own these lights so I can't help you further.

**Reply by themabrennan** [reply]
NOT all lights work as I am finding out. 

Lumary Smart Ultra-thin Panel Light A2 (Yes - Work) 
Lumary Smart Ceiling Light A1 (NO - DO NOT WORK)
